xml/System.Net/IPAddress.xml

```diff
@@ -1954,10 +1986,10 @@
         <Parameter Name="address" Type="System.Net.IPAddress" RefType="out" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="ipString">To be added.</param>
-        <param name="address">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="ipString">The byte span to validate.</param>
+        <param name="address">When this method returns, the <see cref="T:System.Net.IPAddress" /> version of the byte span.</param>
```

Comment:
What happens if the method returns false?

Reply:
it is unchanged.